### PR TITLE
Features/66 sdk headset support

### PR DESCRIPTION
**Pull request by zello-eugene**
#66
#67

Added version info into the manifest and `BuildConfig`.
Updated the javadoc.
Closely related to https://github.com/zelloptt/zello-android-client/pull/5290.

The key change here is continuously playing a silent audio track while the app is running in the background. While this is not perfect, in general it gets the job done. One minor annoyance is pulling the system notification drawer down to expose Zello's notification while having the sample app in the foreground breaks handling of the headset button. The reason being that the system stops sending the key events to app activity. I'm going to let this issue slide for now.

It would appear that the javadoc generating step was updated at some point, resulting in substantial changes in html, js, css etc. content. Please skip all the way down to java/kt files.

### CODE REVIEW

File: zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java [hunk 1]
```diff
@@ -274,24 +243,25 @@ public void onClick(View v) {
...
 		_audio = null;
-
-		Headset.stop();
 	}
 
 	@Override
 	protected void onResume() {
 		super.onResume();
 		Zello.getInstance().leavePowerSavingMode();
+		//Headset.onForeground();
```
Comment (padgrayson91) [inline zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java:257]: Assuming this isn't meant to stay this way - should you remove it entirely?

File: zello-sdk/src/main/java/com/zello/sdk/headset/SilencePlayer.kt [hunk 2]
```diff
@@ -11,6 +11,16 @@ package com.zello.sdk.headset
  */
 interface SilencePlayer {
 
+	/**
+	 * Play the silent audio once.
+	 */
+	fun playOnce()
+
+	/**
+	 * Play the sound audio continuously.
```
Comment (padgrayson91) [inline zello-sdk/src/main/java/com/zello/sdk/headset/SilencePlayer.kt:20]:
```suggestion
	 * Play the silent audio continuously.
```

**Review by padgrayson91 — COMMENTED** [review]
The commented out bit makes me nervous - seems important. Otherwise if this works I'm fine with the structure. I don't have headset handy to test

File: zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java (hunk 1 above)
Comment (zello-eugene) [reply zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java]: Yep, forgot to push the last commit.

**Review by padgrayson91 — APPROVED** [review]

**Pull request merged**